components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -264,7 +271,7 @@ subroutine micro_p3_register()
  
    !! module clubb_intr (AaronDonahue: I think these are for MG only.  Should
    !  we remove?  If so, then we will have to make changes to CLUBB interface.
```

Comment:
I think we can remove this comment now since RELVAR is actually being used.

Reply:
Begs the question about ACCRE_ENHAN, but that may be a question for a later time.

Reply:
@AaronDonahue - what is the question about ACCRE_ENHAN? Is it actually being used anywhere? What is it?

Reply:
@PeterCaldwell , yes, the comment was because CLUBB expected the PBUF variable to exist, so we had to create it in P3, but P3 doesn't do anything with it.  I believe it stands for Accretion Enhancement.